pkg/machine/qemu/machine.go

```diff
@@ -1543,3 +1543,57 @@ func (v *MachineVM) editCmdLine(flag string, value string) {
 		v.CmdLine = append(v.CmdLine, []string{flag, value}...)
 	}
 }
+
+// RemoveAndCleanMachines removes all machine and cleans up any other files associatied with podman machine
+func (p *Provider) RemoveAndCleanMachines() error {
```

Comment:
Why is this function logrus.Error rather then returning it?  Looks like it never returns an error?